e2e_tests/standard/test_cli.py

```diff
@@ -91,6 +96,7 @@ def test_sync_container(self, container_type, environment):
                 last_put_version_f.write(str(last_put_version + 2))
 
             with reinitialize_container(container_sys_id, container_type, project=environment.project) as container:
+                self.stop_synchronization_process(container)
```

Comment:
Why is it necessary here? We could even initializie it read-only mode since it's only for verification.

Reply:
added read-only